contrib/runners/inquirer/tests/unit/test_inquirer_runner.py

```diff
@@ -0,0 +1,153 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+
+import inquirer
+from st2actions.container import service
+from st2common.constants.action import LIVEACTION_STATUS_PENDING
+from st2common.constants.pack import SYSTEM_PACK_NAME
+from st2tests.base import RunnerTestCase
+
+
+mock_liveaction_db = mock.Mock()
+mock_liveaction_db.result = {"response": {}}
+
+mock_action_utils = mock.Mock()
+mock_action_utils.get_liveaction_by_id.return_value = mock_liveaction_db
+
+test_parent_id = '1234567890'
+test_parent = mock.Mock()
+
+mock_action_service = mock.Mock()
+mock_action_service.get_root_liveaction.return_value = test_parent
+
+mock_trigger_dispatcher = mock.Mock()
+
+test_user = 'st2admin'
+
+runner_params = {
+    "users": [],
+    "roles": [],
+    "tag": "developers",
+    "schema": {}
+}
+
+
+class InquiryTestCase(RunnerTestCase):
+
+    def test_runner_creation(self):
+        runner = inquirer.get_runner()
+        self.assertTrue(runner is not None, 'Creation failed. No instance.')
+        self.assertEqual(type(runner), inquirer.Inquirer, 'Creation failed. No instance.')
+
+    @mock.patch('inquirer.TriggerDispatcher', mock_trigger_dispatcher)
```

Comment:
If these mocks are the same for every test cases in this module, you can move the mocks to the class level.  You can then do the mock directly in the mock.patch.object method. Examples: https://github.com/StackStorm/st2/blob/master/contrib/runners/mistral_v2/tests/unit/test_mistral_v2.py#L150

Reply:
Ack, fixed in 479a997fa